Documents/MobileAppActors.md

```diff
@@ -0,0 +1,79 @@
+# Actors: User
+
+## User
+Open the App, Go to Galleries, Select Gallery, Like Images, Share Image, Filter Images, Go to Drawing Screen, Select Canvas, Draw, Tap Drawing Options Menu, Select from Drawing Options Menu, Select Another Canvas,
```

Comment:
I just having names of use cases is not needed for user. General description is enough.

Reply:
I've added another actor -> "Guest", and separated actor descriptions of "Users" and "Guests". List of use cases have been removed. Actors involved for each use case have been added to use cases descriptions.